libraries/RC_Channel/RC_Channel.h

```diff
@@ -175,6 +175,7 @@ class RC_Channel {
         SURFACE_TRACKING =    75, // Surface tracking upwards or downwards
         KILL_IMU1 =          100, // disable first IMU (for IMU failure testing)
         KILL_IMU2 =          101, // disable second IMU (for IMU failure testing)
+        STAND_BY  =          102, // standy by functions
```

Comment:
ah, the enum for STAND_BY should be 76.  I think the KILL_IMU stuff was purpose put high so it wouldn't accidentally be set by users (I'm not totally sure of the reason)